app/filtering.js

```diff
@@ -569,9 +569,7 @@ function registerForDownloadListener (session) {
       webContents.forceClose()
     }
 
-    if (getSetting(settings.DOWNLOAD_ALWAYS_ASK)) {
-      item.setPrompt(true)
-    }
+    item.setPrompt(getSetting(settings.DOWNLOAD_ALWAYS_ASK) || false)
```

Comment:
Normal download, via `.url()` (and I assume content-disposition), was correctly not showing the prompt. But when choosing 're-download' it calls `webContents.downloadUrl` which I assume has a default to show the prompt, and it was not explicitly being set here to `false`.